esx_service/utils/auth_api.py

```diff
@@ -93,10 +93,18 @@ def create_tenant_in_db(name, description, vms, privileges):
         error_info = error_code.generate_error_info(ErrorCode.TENANT_ALREADY_EXIST, name)
         return error_info, None
 
-    error_msg, tenant = auth_mgr.create_tenant(name=name,
-                                                description=description,
-                                                vms=vms,
-                                                privileges=privileges)
+    if name == auth.DEFAULT_TENANT:
```

Comment:
so would you suggest "create_tenant" and "create_default_tenant" APIs, or just put 
```python 
if name == DEFAULT_TENANT_NAME
  uuid = DEFAULT_UUID
else: 
  uuid = generate_uuid
```
into the create_tenant ? (I guess the latter, right ?)